validator.js

```diff
@@ -9,7 +9,6 @@
  *  .catch(error => // Error)
```

Comment:
Why not merge this file into `index.js`?

If it's kept external purely for better maintance, then maybe it'll be good to move into `lib` (or `util` as we have one) folder. Keeping it in root folder implies it's some important public endpoint, when it actually should be seen as _private_